pom.xml

```
@@ -10,7 +10,7 @@
         It provides an object name factory for Dropwizard that adds the support for metric labels.
         Labels are of great value for example on aggregating metrics by Prometheus.
     </description>
-    <version>1.0.4</version>
+    <version>1.0.5</version>
```

```suggestion
    <version>1.1.0</version>
```

According to semantic versioning, I think you should increase minor version, because it is not a bug fix

---

Done